docs/gettinginvolved/cluster-setup.rst

```diff
@@ -199,6 +171,79 @@ Or you can run, for example, the acceptance tests against the created cluster:
      --flocker-version='' \
      flocker.acceptance.obsolete.test_containers.ContainerAPITests.test_create_container_with_ports
 
+==============================
```

Comment:
Cleaning up the cluster configuration was a new section as well, so I thought that setting up the cluster configuration would have to be one as well? I tried to put them as just subsections, but they have overlapping parameters, so sphinx would complain. I am leaving it like that for now. @davidoliverSP2 let me know if you want me to change it to something else, I will need to update the docs again as part of the PR I'm currently working on :)